src/XComponent.Functions/Utilities/SerializationHelper.cs

```diff
@@ -59,11 +59,8 @@ public static object DeserializeObjectFromType(Type objType, object objectToDese
             }
             catch (Exception e)
             {
-                Debug.WriteLine(e);
+                throw new SerializationException($"Couldn't deserialize object from {objType} : {e}");

```

Comment:
do not put the exception in the string, pass it to the constructor

https://msdn.microsoft.com/fr-fr/library/d35hfdat(v=vs.110).aspx